spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/KubernetesClientProperties.java

```diff
@@ -87,6 +92,8 @@
 
 	private String serviceAccountNamespacePath = SERVICE_ACCOUNT_NAMESPACE_PATH;
 
+	private String userAgent;
```

Comment:
We are still not on the same page ;)

I think this should be set to `DEFAULT_USER_AGENT` which then changes the logic above.

Reply:
sorry :( a bit late, but I got your point. thank you for your patience

Reply:
No worries at all.  I am going to hold off on merging it now because we are cutting a release tomorrow and since this is not critical I don't want to introduce anything else at the moment.